load_test.py

```diff
@@ -0,0 +1,96 @@
+import time
```

Comment:
Change the name pf the module from load_test.py to admission_control_overload_test.py

Reply:
changed